tests/testthat/test-brms.R

```diff
@@ -1,144 +1,144 @@
-#       )))
-#
-#       expect_identical(find_predictors(m2, flatten = TRUE), c("Petal.Length", "Sepal.Width", "Species"))
-#       expect_identical(find_predictors(m3), list(conditional = list(conditional = c("treat", "c2"))))
-#       expect_identical(find_predictors(m4), list(conditional = c("child", "camper"), zero_inflated = c("child", "camper")))
-#       expect_identical(find_predictors(m4, effects = "random"), list(random = "persons", zero_inflated_random = "persons"))
-#       expect_identical(find_predictors(m4, flatten = TRUE), c("child", "camper"))
-#     })
-#
-#     test_that("find_response", {
-#       expect_equal(find_response(m1, combine = TRUE), "count")
-#       expect_equal(find_response(m2, combine = TRUE), c(SepalLength = "Sepal.Length", SepalWidth = "Sepal.Width"))
-#       expect_equal(find_response(m3, combine = TRUE), c("r", "n"))
-#       expect_equal(find_response(m1, combine = FALSE), "count")
-#       expect_equal(find_response(m2, combine = FALSE), c(SepalLength = "Sepal.Length", SepalWidth = "Sepal.Width"))
-#       expect_equal(find_response(m3, combine = FALSE), c("r", "n"))
-#       expect_equal(find_response(m4, combine = FALSE), "count")
-#     })
-#
-#     test_that("get_response", {
-#       expect_length(get_response(m1), 236)
-#       expect_equal(ncol(get_response(m2)), 2)
-#       expect_equal(colnames(get_response(m2)), c("Sepal.Length", "Sepal.Width"))
-#       expect_equal(ncol(get_response(m3)), 2)
-#       expect_equal(colnames(get_response(m3)), c("r", "n"))
-#       expect_length(get_response(m4), 250)
-#     })
-#
-#     test_that("find_terms", {
-#       expect_identical(find_terms(m1), list(response = "count", conditional = c("log_Age_c", "log_Base4_c", "Trt"), random = "patient"))
-#       expect_identical(find_terms(m1, effects = "fixed"), list(response = "count", conditional = c("log_Age_c", "log_Base4_c", "Trt")))
-#       expect_null(find_terms(m1, component = "zi"))
-#
-#       expect_identical(
-#         find_terms(m2),
-#         list(
-#           response = c(SepalLength = "Sepal.Length", SepalWidth = "Sepal.Width"),
-#           conditional = list(
-#             SepalLength = c("Petal.Length", "Sepal.Width", "Species"),
-#             SepalWidth = c("Species")
-#         )))
-#
-#       expect_identical(find_terms(m2, flatten = TRUE), c("Sepal.Length", "Sepal.Width", "Petal.Length", "Species"))
-#       expect_identical(find_terms(m3), list(response = c("r", "n"), conditional = c("treat", "c2")))
-#
-#       expect_identical(
-#         find_terms(m4),
-#         list(
-#           response = "count",
-#           conditional = c("child", "camper"),
-#           random = "persons",
-#           zero_inflated = c("child", "camper"),
-#           zero_inflated_random = "persons"
-#         ))
-#
-#       expect_identical(find_terms(m4, flatten = TRUE), c("count", "child", "camper", "persons"))
-#     })
-#
-#   }
-# }
+.runThisTest <- Sys.getenv("RunAllinsightTests") == "yes"
```

Comment:
Maybe we can add a check for Travis environment here, too?

Reply:
yes but I don't know how 😮

Reply:
The last one work though 🎉